hack/update-vendor.sh

```diff
@@ -201,9 +214,15 @@ done
 kube::log::status "go.mod: sorting staging modules"
 # tidy staging repos in reverse dependency order.
 # the content of dependencies' go.mod files affects what `go mod tidy` chooses to record in a go.mod file.
-ls staging/src/k8s.io | sort | xargs -I {} echo "k8s.io/{}" > "${TMP_DIR}/tidy_unordered.txt"
+tidy_unordered="${TMP_DIR}/tidy_unordered.txt"
```

Comment:
not strictly necessary, but we were computing this in at least 3 places, so >WET ==> DRY